mora/service/common.py

```diff
@@ -506,3 +516,36 @@ def is_reg_valid(reg):
     return any([gyldighed_obj.get('gyldighed') == 'Aktiv'
                 for tilstand in reg.get('tilstande', {}).values()
                 for gyldighed_obj in tilstand])
+
+
+def add_bruger_history_entry(employee_uuid, note: str):
+    c = lora.Connector()
+    employee_obj = c.bruger.get(employee_uuid)
+
+    # XXX: We have to make some sort of 'meaningful' change to data to be
+    # able to update the 'note' field - which for now amounts to just
+    # updating the virkning notetekst of gyldighed with a garbage value
+    path = ('tilstande', 'brugergyldighed')
+    gyldighed = get_obj_value(employee_obj, path)[-1]
+    gyldighed['virkning']['notetekst'] = str(uuid.uuid4())
+
+    payload = {
+        'note': note
+    }
+
+    payload = set_object_value(payload, path, [gyldighed])
+    c.bruger.update(payload, employee_uuid)
+
+
+def convert_reg_to_history(reg):
+    return {
+        'user_ref': reg['brugerref'],
```

Comment:
Eventually, we'll want to look up this user object. LoRA has a particular UUID which means ‘anonymous’ — 😒 — which we should preferably detect.

Reply:
Yeah, I agree - this is just more or less copying the approach used for history entries in the old API.